components/textfield/src/vwc-textfield.scss

```diff
@@ -244,3 +244,9 @@ vwc-notched-outline {
 	right: 0;
 	color: var(#{scheme-variables.$vvd-color-neutral-70}) !important;
 }
+
+:host([appearance='ghost'i]) {
```

Comment:
shouldn't this member be `layout` in vivid-2?

Reply:
when I worked on the ghost select we talked about changing it to appearance so I used it here as well :)